### PR TITLE
Log file default icon + cartridgeview update

### PR DESCRIPTION
**Pull request by taurgis**
* Log file now has a default icon (feel free to replace it with something else)
* Cartridgeview now expands to the open file in the editor
* Moved some functions around / refactoring

### CODE REVIEW

**Comment by taurgis** [comment]
* Added .editorconfig for consistency in "formatting"
* Added comments to my code and changed vars to const/let (also did this in extension.ts)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+root = true;
+
+[*]
+indent_style = space
```
Comment (SqrTT) [inline .editorconfig]: Currently, tabs are used...

Comment (taurgis) [inline .editorconfig]: Sorry, I based it on the LogViewer. But since that was spaced you also used spaces there. Will change it.

Comment (taurgis) [inline .editorconfig]: This has been changed.

Comment (SqrTT) [inline .editorconfig]: well, that was my mistake... thanx for fixing.

**Comment by SqrTT** [comment]
thanks for changes

**Pull request merged**